pydantic/_internal/_model_construction.py

```diff
@@ -433,9 +431,11 @@ def generate_model_signature(
                 param_name = field.alias
             else:
                 param_name = field_name
+
             if field_name in merged_params or param_name in merged_params:
                 continue
-            elif not is_valid_identifier(param_name):
+
+            if not is_valid_identifier(param_name):
```

Comment:
Just a bit of code style tweaking. I was struggling to understand what's happening here to cover a missing line.